### PR TITLE
{bio}[GCC/9.3.0] seqtk v1.3

### PR DESCRIPTION
**Pull request by smoors**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node373.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz (skylake_avx512), Python 2.7.5
See https://gist.github.com/0b8897c4d042e99fb49395d209fa9a27 for a full test report.

**Comment by pescobar** [comment]
Test report by @pescobar
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
generoso-c1-s-1 - Linux centos linux 8.2.2004, x86_64, Intel(R) Xeon(R) CPU E5-2667 v3 @ 3.20GHz, Python 3.6.8
See https://gist.github.com/1e466715e9094446db2fd566e98d98d2 for a full test report.

**Comment by smoors** [comment]
@pescobar use `--robot` ?

**Comment by pescobar** [comment]
@smoors sure, I did it by mistake. I am now running it with `--robot` but it's taking a while to build the complete toolchain

**Comment by pescobar** [comment]
Test report by @pescobar
**SUCCESS**
Build succeeded for 9 out of 9 (1 easyconfigs in this PR)
generoso-x-1 - Linux centos linux 8.2.2004, x86_64, Intel(R) Xeon(R) CPU E5-2667 v3 @ 3.20GHz, Python 3.6.8
See https://gist.github.com/ceedb60a4a58435e4ea4d1e93e22d64d for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
node127.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.5
See https://gist.github.com/309e10433cd4cbca3a4b3be5c039c7aa for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
node369.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/2030053ec20dde1952af0cb6ef800cab for a full test report.

File: easybuild/easyconfigs/s/seqtk/seqtk-1.3-GCC-9.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+version = '1.3'
+
+homepage = 'https://github.com/lh3/seqtk/'
+description = """Seqtk is a fast and lightweight tool for processing sequences in the FASTA or FASTQ format.
+ It seamlessly parses both FASTA and FASTQ files which can also be optionally compressed by gzip."""
+
+toolchain = {'name': 'GCC', 'version': '9.3.0'}
+
+github_account = 'lh3'
+source_urls = GITHUB_SOURCE
```
[the remark this answers is not in the document]
Comment (smoors) [reply easybuild/easyconfigs/s/seqtk/seqtk-1.3-GCC-9.3.0.eb:13]: you're right, strange though that the test succeeded for @pescobar and me..

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node377.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/878850ffc7b3c8555beab7593dae567b for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node128.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.5
See https://gist.github.com/7e4308c48372697e974bb2703d56caa5 for a full test report.

**Comment by lexming** [comment]
Going in, thanks @smoors !

**Pull request merged**